sdk/formrecognizer/Azure.AI.FormRecognizer/README.md

```diff
@@ -0,0 +1,104 @@
+# README.md template
```

Comment:
Consider removing this file, filing an issue and adding a doc exclusions to https://github.com/Azure/azure-sdk-for-net/blob/master/eng/.docsettings.yml